test/e2e/cypress/e2e/host_details.cy.js

```diff
@@ -363,6 +368,58 @@ context('Host Details', () => {
     });
   });
 
+  describe('Saptune Summary for this host should be displayed', () => {
+    const { hostName } = selectedHost;
+    const saptuneSummarySelector = '.pt-8';
+
+    const scenarios = [
+      {
+        description: 'should show not installed status',
+        name: 'saptune-uninstalled',
+        data: {
+          packageVersion: 'Not installed',
+          configuredVersion: '-',
+          tuningStatus: '-',
+        },
+      },
+      {
+        description: 'should show version is not supported status',
+        name: 'saptune-unsupported',
+        data: saptuneDetailsDataUnsupportedVersion,
+      },
+      {
+        description:
+          'should show package version, configured version and tuning status',
+        name: 'saptune-compliant',
+        data: saptuneDetailsData,
+      },
+    ];
+
+    scenarios.forEach((scenario) => {
```

Comment:
You could destruct this as `forEach(({ data, description, name}) => {` so you don't need to use dot access